### PR TITLE
Update release notes, contributors for Foreman 3.1 RC2

### PR DESCRIPTION
**Pull request by upadhyeammit**

### CODE REVIEW

File: _includes/manuals/3.1/1.2_release_notes.md [hunk 1]
```diff
@@ -340,6 +350,6 @@ Note that this is only support to run Foreman and Foreman Proxy themselves on EL
 
 We'd like to thank the following people who contributed to the Foreman {{page.version}} release:
 
-<!-- update scripts/committers.rb with the correct versions and dates and fill this in -->
+Adam Cecile, Adam Ruzicka, Adi Abramovich, Alexander Olofsson, Amir Fefer, Amit Upadhye, Andrew Teixeira, Anna Vitova, Antoine Beaupré, Avi Sharvit, Bernhard Suttner, Birkir Freyr, Chris Roberts, Dave Thomas, Dominik Matoulek, Eric D. Helms, Evgeni Golov, Ewoud Kohl van Wijngaarden, Gordon Bleux, Jeremy Lenz, John Mitsch, Jonathon Turel, Justin Sherrill, Kamil Szubrycht, Leos Stejskal, Lukáš Zapletal, Magnus Toneby, Marcel Kühlhorn, Marek Hulán, Maria Agaphontzev, Melanie Corr, Nacho Barrientos, Nadja Heitmann, Oleh Fedorenko, Ondřej Ezr, Ondřej Pražák, Patrick Creech, Rahul Bajaj, Romuald Conty, Ron Lavi, Samir Jha, Shimon Shtein, Shira Maximov, Stejskal Leos, Tim Meusel, Tomer Brisker, Trent Anderson, William Clark, Yifat Makias, aabramov, amtilghman, andrewgdewar, maccelf, rafaelguerra01
```
Comment (melcorr) [inline _includes/manuals/3.1/1.2_release_notes.md]: Adi is twice? 
```suggestion
Adam Cecile, Adam Ruzicka, Adi Abramovich, Alexander Olofsson, Amir Fefer, Amit Upadhye, Andrew Teixeira, Anna Vitova, Antoine Beaupré, Avi Sharvit, Bernhard Suttner, Birkir Freyr, Chris Roberts, Dave Thomas, Dominik Matoulek, Eric D. Helms, Evgeni Golov, Ewoud Kohl van Wijngaarden, Gordon Bleux, Jeremy Lenz, John Mitsch, Jonathon Turel, Justin Sherrill, Kamil Szubrycht, Leos Stejskal, Lukáš Zapletal, Magnus Toneby, Marcel Kühlhorn, Marek Hulán, Maria Agaphontzev, Melanie Corr, Nacho Barrientos, Nadja Heitmann, Oleh Fedorenko, Ondřej Ezr, Ondřej Pražák, Patrick Creech, Rahul Bajaj, Romuald Conty, Ron Lavi, Samir Jha, Shimon Shtein, Shira Maximov, Stejskal Leos, Tim Meusel, Tomer Brisker, Trent Anderson, William Clark, Yifat Makias, amtilghman, andrewgdewar, maccelf, rafaelguerra01
```

**Review by melcorr — CHANGES_REQUESTED** [review]
One small thing.

File: _includes/manuals/3.1/1.2_release_notes.md (hunk 1 above)
Comment (upadhyeammit) [reply _includes/manuals/3.1/1.2_release_notes.md]: Nice catch! Updated the committers script with mapping so next time it won't generate duplicate for Adi

**Review by melcorr — APPROVED** [review]
LGTM Thanks!

**Pull request merged**